Traceback (most recent call last):
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1491, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/ming/Documents/TieComm/main.py", line 175, in <module>
    main(args)
  File "/Users/ming/Documents/TieComm/main.py", line 102, in main
    runner = run(exp_config, env, agent)
  File "/Users/ming/Documents/TieComm/runner/runner.py", line 21, in __init__
    self.lamda = self.args.lamda
AttributeError: 'Namespace' object has no attribute 'lamda'